front/src/components/ProfileView/ProfileBanner.tsx

```diff
@@ -30,13 +39,18 @@ const ProfileBanner: React.FC<IProfileBannerProps> = ({
   profilePicture,
   city,
   genres,
+  capacity,
   withDrawer = false,
   drawerOpened = false,
 }) => {
   const { t } = useTranslation();
   const { editMode, editConfirmed } = useProfileEditMode();
   const canEdit = useLocation().pathname.includes('/auth/profile');
-  const genresToDisplay = loading ? loadingGenres : genres;
+  let genresToDisplay = loading ? loadingGenres : genres;
+  genresToDisplay =
+    genresToDisplay.length > 2
+      ? [genresToDisplay[0], genresToDisplay[1]]
+      : genresToDisplay;
```

Comment:
Is this correctly updating the component when you re-assign genresToDisplay ?

Reply:
Yup, tested this morning